deno/gateway/v10.ts

```diff
@@ -231,6 +244,15 @@ export enum GatewayDispatchEvents {
 	GuildRoleCreate = 'GUILD_ROLE_CREATE',
 	GuildRoleDelete = 'GUILD_ROLE_DELETE',
 	GuildRoleUpdate = 'GUILD_ROLE_UPDATE',
+	GuildScheduledEventCreate = 'GUILD_SCHEDULED_EVENT_CREATE',
+	GuildScheduledEventDelete = 'GUILD_SCHEDULED_EVENT_DELETE',
+	GuildScheduledEventUpdate = 'GUILD_SCHEDULED_EVENT_UPDATE',
+	GuildScheduledEventUserAdd = 'GUILD_SCHEDULED_EVENT_USER_ADD',
+	GuildScheduledEventUserRemove = 'GUILD_SCHEDULED_EVENT_USER_REMOVE',
+	GuildSoundboardSoundCreate = 'GUILD_SOUNDBOARD_SOUND_CREATE',
+	GuildSoundboardSoundDelete = 'GUILD_SOUNDBOARD_SOUND_DELETE',
+	GuildSoundboardSoundsUpdate = 'GUILD_SOUNDBOARD_SOUNDS_UPDATE',
```

Comment:
Does this exist or not, I see no types that reference it

Reply:
Right [here](https://github.com/discordjs/discord-api-types/pull/1113/files/ae2cd5c825dee99934b0ed5e5e605a8aa19aecb2#diff-4c8ded2885f17f9ae37adf936ac51bcf504c3b4e2394c016aae40d4329a96047R1358)? Or are you talking about some other types I missed